partiql-parser/src/parse/partiql.lalrpop

```diff
@@ -645,8 +645,14 @@ ExprPair: ast::ExprPair = {
 
 #[inline]
 FunctionCall: ast::Call = {
+    <func_name:"Identifier"> "(" ")" =>
+        ast::Call{ func_name: ast::SymbolPrimitive{ value: func_name.to_owned() }, args: None },
+     <func_name:"Identifier"> "(" "*" ")" =>
+         ast::Call{ func_name: ast::SymbolPrimitive{ value: func_name.to_owned() }, args: None },
     <func_name:"Identifier"> "("  <args:CommaSep<ExprQuery>> ")" =>
-        ast::Call{ func_name: ast::SymbolPrimitive{ value: func_name.to_owned() }, args }
+        ast::Call{ func_name: ast::SymbolPrimitive{ value: func_name.to_owned() }, args: Some(args) },
+    <func_name:"Identifier"> "(" "DISTINCT" <args:CommaSep<ExprQuery>> ")" =>
```

Comment:
Everywhere that `Distinct` is allowed like this, `All` is too: https://jakewheat.github.io/sql-overview/sql-92-grammar.html#_6_5_set_function_specification.

This likely points to adding a `SetStrategy` or similar AST node enum that we add to `Call` and `CallAgg` (see also `GroupStrategy`) with `All` as default.

Reply:
Makes sense, will look into it further.